update to 1.9.1

giovp opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

giovp committed:
update

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`, or selector on line 24:
    - numpy >=1.22, <3.0  # [py>=39]


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).
 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.
 * ℹ️ top-level output has some malformed specs:
- In section host: `python >=3.9, <3.14
- In section run: `python >=3.9, <3.14, `numpy >=1.22, <3.0, `xarray >=2022.3, <2025.0
Requirement spec fields should always be space-separated to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13793769816](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13793769816). Examine the logs at this URL for more detail.</sub>

giovp committed:
update

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

giovp committed:
update

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).

For **recipe/meta.yaml**:

 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13794188402](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13794188402). Examine the logs at this URL for more detail.</sub>

giovp committed:
update

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

giovp committed:
update sha

giovp committed:
update correct sha

giovp committed:
update maintainers

giovp commented:
@h-vetinari @conda-forge/core I've fixed the CI here, updating to latest version, could you kindly merge this? thank you!
@jdblischak I've added you here as maintainer
@timtreis @melonora @LucaMarconato @giovp are maintainers of the [scverse/spatialdata](https://github.com/scverse/spatialdata) a package that relies on xarray-dataclasses

giovp committed:
update maintainer

jdblischak reviewed (changes requested):
Please rerender the feedstock locally with `conda smithy rerender --commit auto` or comment in this PR with the phrase [@conda-forge-admin, please rerender](https://conda-forge.org/docs/maintainer/infrastructure/#conda-forge-admin-please-rerender)

Commit:
MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.47.0, and conda-forge-pinning 2025.03.11.09.37.43

jdblischak reviewed (approved):
Thanks @giovp!

LucaMarconato commented:
Thanks @giovp, happy to be added as a maintainer.

h-vetinari commented:
> I've added you here as maintainer
> @timtreis @melonora

Can you please confirm you want to be added as maintainers?

melonora commented:
Hey! I can confirm

timtreis commented:
I also confirm, thanks!

h-vetinari reviewed (approved):
Welcome!
